zanshinsdk/client.py

```diff
@@ -966,6 +966,21 @@ def get_organization_scan_target_group(self, organization_id: Union[UUID, str],
         return self._request("GET",
                              f"/organizations/{validate_uuid(organization_id)}/scantargetgroups/"
                              f"{validate_uuid(scan_target_group_id)}").json()
+
+    def iter_scan_target_group_compartments(self, organization_id: Union[UUID, str],
+                                            scan_target_group_id: Union[UUID, str]) -> Iterator[Dict]:
+        """
+        Iterates over the scan targets of a group.
```

Comment:
`Iterates over the compartments of a scan target group` is probably the correct string here, correct? Looks like you copied this docstring from elsewhere and forgot to update the description.